docker/base/Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM ministryofjustice/ruby:2.5.1
 
+# https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash -
+
 RUN apt-get update && apt-get install -y nodejs postgresql-contrib libpq-dev
```

Comment:
do we need to remove nodejs from this line?

Reply:
From the installation instructions look like we should do both, I guess the first one is configuring the sources list 

![image](https://user-images.githubusercontent.com/381495/60339325-6338e180-99a0-11e9-9138-eabf05bce049.png)

Reply:
But surely we should rewrite in Ruby.....

Reply:
^^^ I am being serious, just in case that comes across as snark

Reply:
maybe its not even needed. it might have to do with asset minification, but I'd have to check